src/lib/openapi/spec/advanced-playground-request-schema.ts

```diff
@@ -0,0 +1,48 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { ALL } from '../../types/models/api-token';
+import { sdkContextSchema } from './sdk-context-schema';
+
+export const advancedPlaygroundRequestSchema = {
+    $id: '#/components/schemas/advancedPlaygroundRequestSchema',
+    description:
+        'Data for the playground API to evaluate toggles in advanced mode with environment and context multi selection',
+    type: 'object',
+    required: ['environments', 'context'],
+    properties: {
+        environments: {
+            type: 'array',
+            items: { type: 'string' },
+            example: ['development', 'production'],
+            description: 'The environments to evaluate toggles in.',
+        },
+        projects: {
+            description: 'A list of projects to check fro toggles in.',
+            oneOf: [
+                {
+                    type: 'array',
+                    items: { type: 'string' },
+                    example: ['my-project'],
+                    description: 'A list of projects to check for toggles in.',
+                },
+                {
+                    type: 'string',
+                    enum: [ALL],
+                    description: 'Check toggles in all projects.',
+                },
+            ],
+        },
+        context: {
+            description: 'The context to use when evaluating toggles',
+            $ref: sdkContextSchema.$id,
```

Comment:
If I remember correctly, then using a $ref should propagate the example that exists on the referenced schema. Can you double check that that works correctly in swagger UI?

Reply:
meta-schema-rules.test.ts was failing w/o this explicit description. No idea why

Reply:
Ah, that'll be because it expects all properties to have the 'description' property. However, if the $ref property propagates the description, then we might be able to update the tests. However, I don't think that it propagates descriptions.

Reply:
Yes, test is broken. I will create a followup PR that fixes the test